google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java

```diff
@@ -99,6 +108,12 @@
     panel.setPreferredSize(new Dimension(800, 600));
 
     projectSelector.loadActiveCloudProject();
+
+    if (ServiceManager.getService(PluginInfoService.class).shouldEnable(GctFeature.BOM)) {
+      populateBomVersions();
+    } else {
+      hideBomUI();
```

Comment:
When the feature is off, and components are hidden, does the panel adjust itself properly, i.e., there is no large empty spaces?

Reply:
yes. Ive confirmed it